helpers.lua

```diff
@@ -471,6 +484,8 @@ function get_data_for(view)
         or view == "edit.html"
         or view == "password.html"
         or view == "ynhpanel.json" then
```

Comment:
Hum, second thought : let's do what you proposed first : take ynhpanel.json out of the "needs cache refresh" list. Since it does not need aliases and forwards, it is not that much important if these info are out of date.